src/main/java/seedu/address/commons/util/ListUtil.java

```
@@ -0,0 +1,40 @@
+package seedu.address.commons.util;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Utility methods related to Lists
+ */
+public class ListUtil {
+
+    /**
+     * Returns a sublist of items from {@code list} specified by {@code indices}.<br>
+     * - Original list order is maintained in the extracted list.<br>
+     * - Duplicate indices are ignored.<br>
+     */
+    public static <T> List<T> subList(List<T> list, Integer... indices) {
+        return subList(list, Arrays.asList(indices));
+    }
+
+    /**
+     * Returns a sublist of items from {@code list} specified by {@code indices}.<br>
+     * - Original list order is maintained in the extracted list.<br>
+     * - Duplicate indices are ignored.<br>
+     */
+    public static <T> List<T> subList(List<T> list, Collection<Integer> indices) {
```

I think we should make `subList` take a Set of indices instead, rather than silently ignoring duplicates. There are a few reasons why:

* The type signature of the method should describe its functionality as succinctly as possible. This allows readers of code to quickly grasp the responsibility of the method by looking at its input and output types.

* From a logical perspective, it makes sense to say "I want a sublist with items 0, 1 and 2", but it does not make sense to say "I want a sublist with items 0 and 0" [1]. Such a query is totally nonsense.

* This would force callers to think long and hard about whether they should be passing a collection of indices with duplicates in the first place. Passing a collection with duplicates may actually indicate a bug in the program (e.g. in test code, having test data with duplicate indices)

[1] An alternative interpretation of `subList()` could actually be to directly map each index to its respective item of the list, respecting the ordering and any duplicates. Taking a `Set` instead of a `Collection` would help readers avoid assuming this wrong interpretation.